packages/launch/src/focused-launch/index.tsx

```diff
@@ -46,11 +46,30 @@ const FocusedLaunch: React.FunctionComponent = () => {
 	const planProductIdFromCart = usePlanProductIdFromCart();
 	const { setPlanProductId } = useDispatch( LAUNCH_STORE );
 
+	// Once there is a selected plan, do not retrieve plan from from cart anymore.
+	const canUsePlanProductIdFromCart = React.useRef( false );
```

Comment:
For the variable name to make sense we should probably initialize it with `true`, right? 
Then of course, update the following occurrences.